azavea/mos-energy-benchmark
# Fix autocomplete

## Overview
This issue was to diagnose autocomplete, but once I got to the point where I was able to understand the issue well enough to diagnose it which involved get a development environment up and running, fixing it and making setup reproducible was not much more work so I went ahead and did that!

## Demo
![mos_fix_autocomplete](https://user-images.githubusercontent.com/2926237/116300409-dd894700-a76c-11eb-9176-0ffe6776f973.gif)

## Notes
This fixes autocomplete for addresses specifically, but it doesn't fix it for Philadelphia building ids (building ids still work in that putting one in the search text input and pressing enter still zooms to that building, however). It seems like the data returned by the API has changed out from under us and investigating that was beyond the scope of this issue. Perhaps that functionality has been broken for a long while (last touched in 2014; see 85887ea2). This at least allows us to run and (hopefully) deploy the project with one less issue.

The build is also breaking. That probably wouldn't be too hard to fix now that this is containerized but I didn't try since I used up all the time in my timebox.

## Testing Instructions
* Follow README to build and run project
* Make sure autocomplete now works for addresses

Closes https://github.com/azavea/mos-energy-benchmark/issues/281
